server/mdm/maintainedapps/installers.go

```diff
@@ -10,13 +10,15 @@ import (
 	"path"
 	"time"
 
-	"github.com/fleetdm/fleet/v4/pkg/fleethttp"
 	"github.com/fleetdm/fleet/v4/server/contexts/ctxerr"
 	"github.com/fleetdm/fleet/v4/server/fleet"
 )
 
+// InstallerTimeout is the timeout duration for downloading and adding a maintained app.
+const InstallerTimeout = 15 * time.Minute
```

Comment:
Can we define this in the same file where we use it?

Reply:
Fair point! I think I added it here before passing the client as a param.